extras/1f3fb.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64" enable-background="new 0 0 64 64"><circle fill="#ffe1bd" cx="32" cy="32" r="30"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 36 36"><path fill="#F7DECE" d="M33 30c0 1.657-1.343 3-3 3H6c-1.657 0-3-1.343-3-3V6c0-1.657 1.343-3 3-3h24c1.657 0 3 1.343 3 3v24z"/></svg>
```

Comment:
Any reason that you keep these `svg` files in `extras`? They came from Twemoji and exists in your zip file.

Reply:
I thought they were part of EmojiOne, since I didn't see that part of `LICENSE.md` originally.